README.rst

```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
 ==========================================
 
 The Atomic Developer Bundle (ADB) is a prepackaged development
-environment filled with production-grade pre-configured tools that makes
-container developer's lives easier.  The ADB supports the development
+environment filled with production-grade, pre-configured tools, that makes
+container developers' lives easier.  The ADB supports the development
 of multi-container applications against different technologies and
 orchestrators while providing a path that promotes best practices.
 
 As a container developer, you want to use the ADB for these reasons:
 
 * Pre-Configured: You don't have to spend time building an environment
```

Comment:
Could you make point title as bold? 
i.e. `Pre-Configured`, `Multiple Environment Support` etc

These point's titles are not visible when looking that section as whole.